base/exports.jl

```diff
@@ -1197,6 +1197,9 @@ export
     init_worker,
     interrupt,
     isready,
+    IC_MASTER_SLAVE,
+    IC_ALL_TO_ALL,
+    IC_CUSTOM,
```

Comment:
I haven't used enums in Julia yet so I haven't thought much about what is the right way to use them. It just seems a bit much to export new names when they are only used in his function and at the same time `Multiprocessing.IC_MASTER_SLAVE` or `Parallel.IC_MASTER_SLAVE` is a bit long.